projects/1.18/assets/miniature-power-plant/miniaturepowerplant/lang/zh_cn.json

```
@@ -0,0 +1,66 @@
+{
+  "itemGroup.miniaturepowerplant":  "微型发电站",
+  "block.miniaturepowerplant.carrier": "发电厂",
+
+  "item.miniaturepowerplant.depletable.desc.depletion": "已消耗: %s%%",
+  "item.miniaturepowerplant.depletable.desc.depleted": "已耗尽",
+  "item.miniaturepowerplant.depletable.desc.new": "全新的",
+
+  "item.miniaturepowerplant.solar_module": "太阳能模块",
+  "item.miniaturepowerplant.solar_module_stone": "太阳能模块(石)",
+  "item.miniaturepowerplant.solar_module_gold": "太阳能模块(金)",
+  "item.miniaturepowerplant.solar_module_iron": "太阳能模块(铁)",
+  "item.miniaturepowerplant.solar_module_lapis": "太阳能模块(青金石)",
+  "item.miniaturepowerplant.solar_module_redstone": "太阳能模块 (红石)",
+  "item.miniaturepowerplant.solar_module_diamond": "太阳能模块(钻石)",
+  "item.miniaturepowerplant.solar_module_netherite": "太阳能模块(下界合金)",
+  "item.miniaturepowerplant.solar_module.production.brightness": "当前亮度",
+  "item.miniaturepowerplant.solar_module.penalty.rain": "下雨",
+  "item.miniaturepowerplant.solar_module.penalty.thunder": "打雷",
+  "item.miniaturepowerplant.solar_module.desc.energy_multiplier": "能量产生: %s%%",
+
+  "item.miniaturepowerplant.temperature_module_basic": "温度模块",
+  "item.miniaturepowerplant.temperature_module.production.adjusted_temperature": "当前温度",
+  "item.miniaturepowerplant.temperature_module.penalty.rain": "下雨",
+
+  "item.miniaturepowerplant.water_module_basic": "水模块",
+  "item.miniaturepowerplant.water_module.production.water_level": "水位@ %s",
```

```suggestion
  "item.miniaturepowerplant.water_module.production.water_level": "水位处于%s",
```
@ = at（猜测）
懒得打字属于是（）